.eslintrc.js

```diff
@@ -0,0 +1,79 @@
+module.exports = {
+	"root": true,
+	"plugins": ["node", "prettier"],
+	"extends": ["eslint:recommended", "plugin:node/recommended"],
+	"env": {
+		"node": true,
+		"es6": true,
+		"jest": true
+	},
+	"parserOptions": { "ecmaVersion": 2017 },
+	"rules": {
+		"quote-props": ["error", "as-needed"],
+		"no-dupe-keys": "error",
+		"quotes": ["error", "double"],
+		"no-undef": "error",
+		"no-extra-semi": "error",
+		"semi": "error",
+		"no-template-curly-in-string": "error",
+		"no-caller": "error",
+		"yoda": "error",
+		"eqeqeq": "error",
+		"global-require": "off",
+		"brace-style": "error",
+		"key-spacing": "error",
+		"space-in-parens": ["error", "never"],
+		"space-infix-ops": "error",
+		"indent": ["error", "tab", { "SwitchCase": 1 }],
+		"no-extra-bind": "warn",
+		"no-empty": "off",
+		"no-multiple-empty-lines": "error",
+		"no-multi-spaces": "error",
+		"no-process-exit": "off",
+		"no-trailing-spaces": "error",
+		"no-use-before-define": "off",
+		"no-unused-vars": ["error", { "args": "none" }],
+		"no-unsafe-negation": "error",
+		"no-loop-func": "warn",
+		"space-before-function-paren": ["error", "never"],
+		"space-before-blocks": "error",
+		"object-curly-spacing": ["error", "always"],
+		"keyword-spacing": ["error", {
+			"after": true,
+			"overrides": {
+				"const": { "after": true },
+				"try": { "after": true },
+				"catch": { "after": true },
+				"if": { "after": true },
+				"else": { "after": true },
+				"throw": { "after": true },
+				"case": { "after": true },
+				"return": { "after": true },
+				"finally": { "after": true },
+				"do": { "after": true }
+			}
+		}],
+		"no-console": "off",
+		"valid-jsdoc": "error",
+		"node/no-unsupported-features": ["error", { "version": 4 }],
+		"node/no-deprecated-api": "error",
+		"node/no-missing-import": "error",
+		"node/no-missing-require": [
+			"error",
+			{
+				"allowModules": [
+					"webpack"
+				]
+			}
+		],
+		"node/no-unpublished-bin": "error",
+		"node/no-unpublished-require": "error",
+		"eol-last": ["error", "always"],
+		"newline-per-chained-call": "off",
+		"node/process-exit-as-throw": "error",
+		"prettier/prettier": ["error", {
+			"useTabs": true,
+			"bracketSpacing": false
```

Comment:
let me double check, I was getting crazy errors of compatibility between eslint and prettify, did some tests, this might have squeezed in.